website/docusaurus.config.ts

```diff
@@ -295,6 +295,7 @@ const config: Config = {
                     remarkPlugins: [[pluginNpm2Yarn, { sync: true }]],
                     docItemComponent: '@theme/ApiItem',
                     sidebarPath: './sidebars.ts',
+                    showLastUpdateTime: true,
```

Comment:
@alvinometric @nnennandukwe wdyt of having the last updated date shown at the bottom of each page?

Reply:
I'm hesitant, I like seeing it but at the same time I don't want to give the impression that some of our content is outdated

Front-end tech in particular moves so fast that 1-year-old content is often out of date, which isn't the case with our SDKs and our tutorials

Reply:
You're right, maybe in a  few months if/when I've had the chance to make updates to the majority of the content, then we can reconsider